cdap-ui/app/features/tracker/templates/integrations.html

```diff
@@ -0,0 +1,154 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="integrations-container">
+  <div class="row integration-item">
```

Comment:
It will be easier for us to manager if we separate out bootstrap's css class and our own css class. Its always good practice to not overlap them.

```
<div class="row">
  <div class="intergration-item">
  </div>
</div>
```
may be?